contracts/libraries/Account.sol

```diff
@@ -429,7 +425,7 @@ library Account {
             (, int256 vQuoteAmountSwapped) = account.tokenPositions.swapToken(
                 account.id,
                 poolId,
-                IClearingHouseStructures.SwapParams(tokensToTrade, sqrtPriceLimit, false, true),
+                IClearingHouseStructures.SwapParams(tokensToTrade, sqrtPriceLimit, false, true, false),
```

Comment:
Using solidity object notation for passing function parameters can make these false true false more readable.

```suggestion
                IClearingHouseStructures.SwapParams({amount: tokensToTrade, sqrtPriceLimit: sqrtPriceLimit, isNotional: false, isPartialAllowed: true, shouldSettleProfit: false}),
```